src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$catalog-max-width: 944px;
+$card-hover-color: #34568b;
+$card-hover-scale: 120%;
+$card-height: 300px;
+$card-width: 200px;
+$header-logo-height: 40px;
+$header-height: 60px;
+$header-hover-color: #00acdc;
+$nav-item-color: #060b35;
+$transition-duration-time: 300ms;
```

Comment:
do you really need all these variables? maybe it would be great to add only variables that are used in several places (e.g. `$transition-duration-time`)